index.html

```diff
@@ -19,22 +19,30 @@
         <link
             href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto:wght@500;700&display=swap"
             rel="stylesheet">
+            <script src="script.js"></script>
 </head>
 
 <body>
 <main>
     <!----------navbar setion------------------->
     <header>
     <nav class="nav-container">
-        <a class="Johnd" href=""></a>
-        <a class="menu" href="#"></a>
-        <ul>
-            <li><a href="">Portfolio</a></li>
-            <li><a href="">About</a></li>
-            <li><a href="">Contact</a></li>
-            <li><a href=""><img src="/images/msg.png"></a></li>
+       
+        <a class="Johnd" href="#"></a>
+        <a class="menu" href="#" onclick="displayPopUp()"></a>
+        <ul class="menu-items">
+            <li><a href="#my-works">Portfolio</a></li>
+            <li><a href="#about-me">About</a></li>
+            <li><a href="#contact-me">Contact</a></li>
+            <li><a href="#"><img src="./images/msg.png"></a></li>
```

Comment:
- [ ] as the previous reviewer asked  it would be better if you use the same font weight for the links in the menu mobile in order to match the Design
- yours 
 
<img width="198" alt="Screenshot 2023-02-06 at 15 36 49" src="https://user-images.githubusercontent.com/61790587/217000028-128fb519-908e-41d8-a58f-6ec8e43e633f.png">
- Design 
<img width="154" alt="Screenshot 2023-02-06 at 15 36 34" src="https://user-images.githubusercontent.com/61790587/217000073-df57f252-2078-413f-a7d9-043ffaa57fb9.png">

Reply:
font weight used as per figma